src/api/middleware/validators/offer.js

```diff
@@ -452,10 +454,30 @@ export const setDefaultValuesCreate = (req, res, next) => {
 };
 
 export const get = useExpressValidators([
-    query("offset", ValidationReasons.DEFAULT)
+    query("lastOfferId", ValidationReasons.DEFAULT)
         .optional()
-        .isInt({ min: 0 }).withMessage(ValidationReasons.INT)
-        .toInt(),
+        .custom(isObjectId).withMessage(ValidationReasons.OBJECT_ID).bail()
+        .custom(existingOfferId)
+        .custom(async (offerId, { req }) => {
```

Comment:
if this validator is only for the `OFFER_NOT_MATCHING_CRITERIA` check, then we can probably remove it